content/ko/docs/topics/charts.md

```diff
@@ -0,0 +1,1123 @@
+---
+title: "차트"
+description: "차트 형식을 설명하고, 헬름으로 차트를 빌드하기 위한 기본지침을 제공한다."
+aliases: [
+  "docs/developing_charts/",
+  "developing_charts"
+]
```

Comment:
Please remove these aliases. These only apply to the English version, and if merged, this would send readers of English documentation to Korean pages.

Reply:
Yes, I  will remove aliases as well as also in future translations.